README.md

```diff
@@ -17,22 +17,21 @@ The Kubeflow pipelines service has the following goals:
 
 ## Installation
 
-* Install Kubeflow Pipelines from choices described in [Installation Options for Kubeflow Pipelines](https://www.kubeflow.org/docs/pipelines/installation/overview/).
+* Kubeflow Pipelines can be installed as part of the [Kubeflow Platform](https://www.kubeflow.org/docs/started/installing-kubeflow/#kubeflow-platform). Alternatively you can deploy [Kubeflow Pipelines (V2)](https://www.kubeflow.org/docs/components/pipelines/operator-guides/installation/) as a standalone service.
```

Comment:
nit: I'd suggest remove 'V2' here, the standalone deployment instruction applies to all versions.

Reply:
Updated and commits squashed. Thanks @chensun